cmd/internal/find/walker.go

```diff
@@ -25,49 +25,63 @@ func newWalker(r parser.Result, conn client.Client) *walker {
 	}
 }
 
-func (w *walker) Walk(e types.Entry) {
-	w.walk(e, 0)
+// Walk returns true if the walk is successful (i.e. does not
+// have any errors), false otherwise.
+func (w *walker) Walk(path string) bool {
```

Comment:
There's enough code here that it's worth having some unit tests. I'll do that after this PR.